backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/slider/internal/BpkRangeSliderImpl.kt

```diff
@@ -108,44 +117,69 @@ private fun SlideRangeLabel(
     interactionSource: MutableInteractionSource,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .semantics { stateDescription = label }
-            .padding(bottom = BpkSpacing.Xl),
-        verticalArrangement = Arrangement.Center,
-        horizontalAlignment = Alignment.CenterHorizontally,
-    ) {
-        BpkText(
-            text = label,
-            color = BpkTheme.colors.textPrimaryInverse,
-            style = BpkTheme.typography.label2,
-            textAlign = TextAlign.Center,
+    LabelLayout(modifier) {
+        Box(
             modifier = Modifier
                 .background(
                     color = BpkTheme.colors.coreAccent,
                     shape = FlareShape(
-                        borderRadius = BpkBorderRadius.Sm,
-                        flareHeight = BpkSpacing.Sm,
+                        borderRadius = BorderRadius,
+                        flareHeight = FlareHeight,
                         pointerDirection = BpkFlarePointerDirection.Down,
                     ),
                 )
-                .padding(top = BpkSpacing.Sm)
-                .padding(bottom = BpkSpacing.Md)
-                .padding(horizontal = BpkSpacing.Md)
-                .semantics { invisibleToUser() },
-
-        )
-        Spacer(
-            modifier = Modifier.height(BpkSpacing.Sm),
-        )
+                .padding(bottom = FlareHeight)
+                .height(BpkSpacing.Lg)
+                .padding(horizontal = BpkSpacing.Md),
+            contentAlignment = Alignment.Center,
+        ) {
+            BpkText(
+                text = label,
+                color = BpkTheme.colors.textPrimaryInverse,
+                style = BpkTheme.typography.label2,
+                textAlign = TextAlign.Center,
+                maxLines = 1,
+                modifier = Modifier
+                    .semantics { invisibleToUser() },
+            )
+        }
         SliderDefaults.Thumb(
+            modifier = Modifier.padding(top = LabelOffset)
+                .semantics { stateDescription = label },
             interactionSource = interactionSource,
             colors = sliderColors(),
             enabled = enabled,
         )
     }
 }
 
+@Composable
+private fun LabelLayout(
```

Comment:
had to create a custom layout here to ensure that the label itself wasn't taking up any space, otherwise the range slider track will jump around constantly as it tries to recalculate the width as the label width changes